docs/usermanual.md

```diff
@@ -194,6 +194,15 @@ npm run
 
```

Comment:
CHANGES_NEXT_RELEASE entry as in PR https://github.com/telefonicaid/iotagent-json/pull/606, pls.

Reply:
Fixed Fixed 6f9106f